server/src/main/java/org/candlepin/model/activationkeys/ActivationKeyPool.java

```diff
@@ -56,16 +54,12 @@
     private String id;
 
     @ManyToOne
-    @ForeignKey(name = "fk_activation_key_pool_k")
```

Comment:
We had a discussion a while ago with some of the other folks, and we decided it is better to clean these 2 types off annotations up when getting the chance (Index & ForeignKey), because they aren't really used (hibernate uses these only if you auto-create the schema, to apply them, and we use liquibase for schema changes instead), and they are outdated / not in sync with what the actual schema indices/foreign keys look like.